docs/example_config.yaml

```diff
@@ -11,38 +11,36 @@ server:
     cert_key: cert
     key_key: key
     ca_key: ca
-ntoken:
+nToken:
   athenz_domain:  _athenz_domain_
```

Comment:
It looks like there's something in this file that needs to be fixed.

Reply:
fixed in: 34243eb